src/test/resources/bandits/rac-experiments-bandits-beta.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
don't forget to add this to the shared github repo

Reply:
Yeah that's the plan once this stuff solidifies and we merge the fork!